command/operator_gossip.go

```diff
@@ -0,0 +1,30 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type OperatorGossipCommand struct {
+	Meta
+}
+
+func (*OperatorGossipCommand) Help() string {
+	helpText := `
+	Usage: nomad operator gossip [options]
+	
+	This command is accessed by using one of the subcommands below.
+	`
```

Comment:
When running the command locally, the output format seemed slightly off. The whitespace suggestion fixes that and the change to the usage keeps more consistency with other commands and correctly shows subcommand expectation. 

```suggestion
Usage: nomad operator gossip <subcommand> [options] [args]
	
  This command is accessed by using one of the subcommands below.
`
```